docs/Troubleshooting.md

```diff
@@ -216,7 +237,11 @@ setTimeout(() => {
 
 When Jest runs your test to collect the `test`s it will not find any because we have set the definition to happen asynchronously on the next tick of the event loop.
 
-_Note:_ This means when you are using `test.each` you cannot set the table asynchronously within a `beforeEach` / `beforeAll`.
+:::note
```

Comment:
Instead of admonition, simply keep this inline: "...next tick of the event loop. This means when..."